grobid-core/src/main/java/org/grobid/core/jni/PythonEnvironmentConfig.java

```diff
@@ -60,16 +77,17 @@ public static PythonEnvironmentConfig getInstanceForVirtualEnv(String virtualEnv
         }
         if (StringUtils.isEmpty(virtualEnv)) {
             virtualEnv = activeVirtualEnv;
-        } else if(StringUtils.isEmpty(activeVirtualEnv)) {
```

Comment:
This didn't seem to make sense to me. What wasn't working without setting `activeVirtualEnv`? It seems `isActive` isn't actually used, therefore I am not sure what effect it should have.

Reply:
no problem with that, let's remove it